Logging to shared log directory: /tmp/test_logs//1331873035
only running test 6
excepting tests: 
mft01: Checking config lustre mounted on /mnt/lustre
Checking servers environments
Checking clients mft01 environments
Using TIMEOUT=20
disable quota as required
debug=0x33f0404
subsystem_debug=0xffb7e3ff
debug_mb=2
setting all flavor to null
already have total 5 null connections
/mnt/lustre/d0.replay-ost-single
stripe_count:   1 stripe_size:    1048576 stripe_offset:  0 
.......

== replay-ost-single test 6: Fail OST before obd_destroy == 06:43:56 (1331873036)
Waiting for orphan cleanup...
Waiting for destroy to be done...
1280+0 records in
1280+0 records out
5242880 bytes (5.2 MB) copied, 0.622363 s, 8.4 MB/s
/mnt/lustre/d0.replay-ost-single/f6
lmm_stripe_count:   1
lmm_stripe_size:    1048576
lmm_stripe_offset:  0
	obdidx		 objid		objid		 group
	     0	           162	         0xa2	             0

fail_loc=0x80000119
before: 325400 after_dd: 320280
Failing ost1 on node mft01
Stopping /mnt/ost1 (opts:) on mft01
affected facets: ost1
Failover ost1 to mft01
06:44:26 (1331873066) waiting for mft01 network 900 secs ...
06:44:26 (1331873066) network interface is UP
Starting ost1: -o loop  /tmp/lustre-ost1 /mnt/ost1
debug=0x33f0404
subsystem_debug=0xffb7e3ff
debug_mb=2
Started lustre-OST0000
affected facets: ost1
mft01: *.lustre-OST0000.recovery_status status: COMPLETE
Can't lstat /mnt/lustre/d0.replay-ost-single/f6: No such file or directory
Waiting for orphan cleanup...
Waiting for destroy to be done...
before: 325400 after: 325400
Resetting fail_loc on all nodes...done.
PASS 6 (34s)
.== replay-ost-single replay-ost-single.sh test complete, duration 35 sec == 06:44:30 (1331873070)
